config/routes.rb

```diff
@@ -10,12 +10,10 @@
 
   # Icon redirects
   get "/favicon.ico", to: "icon_redirects#show"
-  get "/apple-touch-icon.png", to: "icon_redirects#apple_60_60"
+  get "/apple-touch-icon.png", to: "icon_redirects#apple_touch_icon"
+  get "/apple-touch-icon-180x180.png", to: "icon_redirects#show"
+  get "/apple-touch-icon-167x167.png", to: "icon_redirects#show"
   get "/apple-touch-icon-152x152.png", to: "icon_redirects#show"
```

Comment:
Smarter folk than me might want to weigh in here with thoughts about us suddenly 404-ing all these old icons.  Should we leave the routes in and redirect to the closest current size that we do have an asset for?